features/mesh/services/Item.feature

```diff
@@ -2,29 +2,31 @@ Feature: mesh / services / item
   Background:
     Given the CSS selectors
       | Alias                  | Selector                                         |
+      | config-tab             | #service-config-view-tab a                       |
       | data-plane-proxies-tab | #service-data-plane-proxies-view-tab a           |
       | item                   | [data-testid='data-plane-collection'] tbody tr   |
       | input-search           | [data-testid='k-filter-bar-filter-input']        |
       | button-search          | [data-testid='k-filter-bar-submit-query-button'] |
       | button-clear-search    | [data-testid="k-filter-bar-clear-query-button"]  |
 
-  Scenario Outline: Shows Data Plane Proxies for service type <ServiceType>
+  Scenario Outline: Shows correct tabs for service type <ServiceType>
     Given the URL "/meshes/default/service-insights/firewall-1" responds with
       """
         body:
           serviceType: <ServiceType>
       """
 
     When I visit the "/meshes/default/services/firewall-1/overview" URL
-    Then the "$data-plane-proxies-tab" element <ExistsAssertion>
+    Then the "$config-tab" element <ConfigTabExistsAssertion>
+    Then the "$data-plane-proxies-tab" element <DppTabExistsAssertion>
 
     Examples:
-      | ServiceType       | ExistsAssertion |
-      | ~                 | exists          |
-      | internal          | exists          |
-      | gateway_builtin   | exists          |
-      | gateway_delegated | exists          |
-      | external          | doesn't exist   |
+      | ServiceType       | ConfigTabExistsAssertion | DppTabExistsAssertion |
+      | ~                 | doesn't exist            | exists                |
+      | internal          | doesn't exist            | exists                |
+      | gateway_builtin   | doesn't exist            | exists                |
+      | gateway_delegated | doesn't exist            | exists                |
+      | external          | exists                   | doesn't exist         |
```

Comment:
I'd really rather avoid these types of "put the assertion in the table" if possible, they take extra to figure out what is being tested, although I can see why you did it here, it wouldn't be great doing a repeated test for each of these cases.

🤔 

What about if we had 2 ScenarioOutlines? If I'm not mistaken there are two groups here? A group where the config tab exists and the dataplane tab doesn't, and another group where its the opposite. Plus there is only one outlier, otherwise they'd all be the same.

Bit of a nit this, but half the reason for the Gherkin tests is to try and write them in a way which makes them super easy to see whats being tested. All in all not super necessary to change here/now, but up to you.

Reply:
Fair enough. I changed this.